### PR TITLE
Update Istio Operator

### PR DESCRIPTION
**Pull request by rschoebel**
from version 1.12.2 -> 1.18.1

### CODE REVIEW

**Review by medyagh — COMMENTED** [review]
[rschoebel](https://github.com/rschoebel) do u mind posting Before/After this PR  output  to ensure this is not breaking ?

**Comment by rschoebel** [comment]
Screenshot from 2023-07-19 04-26-52

i hope this is enough

File: pkg/minikube/assets/addons.go [hunk 1]
```diff
@@ -275,7 +275,7 @@ var Addons = map[string]*Addon{
 			"istio-operator.yaml",
 			"0640"),
 	}, false, "istio-provisioner", "3rd party (Istio)", "", "https://istio.io/latest/docs/setup/platform-setup/minikube/", map[string]string{
-		"IstioOperator": "istio/operator:1.12.2@sha256:42c7609872882cb88728a1592561b4046dac6d05b6002cbdc815b84c86a24f08",
+		"IstioOperator": "istio/operator:1.18.1@sha256:86929072495a0c8128802587208780649d61394167579eea02786d91090d0c66",
```
Comment (spowelljr) [inline pkg/minikube/assets/addons.go:278]: When I pull the image I get a different SHA than you have listed
```
docker pull istio/operator:1.18.1
1.18.1: Pulling from istio/operator
c39e4b8c44d6: Pull complete 
ec96bb23360c: Pull complete 
4658f4a514c5: Pull complete 
28d661d108f2: Pull complete 
434155d50110: Pull complete 
Digest: sha256:9ae5aa7508374da4c8052389c2aef521ba360ee7bc54f5e1480b7fe4b33c6e48
```

**Review by spowelljr — CHANGES_REQUESTED** [review]

File: pkg/minikube/assets/addons.go (hunk 1 above)
Comment (rschoebel) [reply pkg/minikube/assets/addons.go:278]: I tried it myself, it seems like the have 2 checksums for the the same image

but dockerhub only shows one, 
https://hub.docker.com/layers/istio/operator/1.18.1/images/sha256-86929072495a0c8128802587208780649d61394167579eea02786d91090d0c66?context=explore

```
docker pull istio/operator:1.18.1@sha256:86929072495a0c8128802587208780649d61394167579eea02786d91090d0c66
docker pull istio/operator:1.18.1
docker inspect istio/operator:1.18.1
```

**Pull request closed without merging**